Firmware/cmdqueue.cpp

```diff
@@ -479,9 +479,10 @@ void get_command()
             allow_when_stopped = true;
 
         // Handle the USB timer
-        if ((*cmd_start == 'G') && !(IS_SD_PRINTING))
+        if ((*cmd_start == 'G') && !(IS_SD_PRINTING)) {
             usb_timer.start();
-
+            printer_state = 3; //set printer state busy printing to hide LCD menu and report correctly M862.7 Q while USB printing
```

Comment:
I think it would be nice to use `enum` or` enum class` instead of magic numbers.

Reply:
Agreed, but i would like to change that later to cover more.